### ISSUE TITLE
[Security] De-scope Moped CI access permissions

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#11150]
1. Identify the minimum AWS permissions needed for the Moped CI
2. Adjust the AWS user permissions for the Moped CI users to match ☝️

**Comment by frankhereford** [comment]
Consider the test commissioning system and isolate it into its own user.

**Comment by mddilley** [comment]
After looking at IAM Access Advisor, I've updated the IAM user named `atd-moped-ci` with permissions that cover:
- Zappa deployment by attaching the `atd-zappa` policy
- SQS
- CloudFront
- Secrets Manager

This is a drastic decrease in permissions, but permissions could be further reduced. That would be a good topic for the next security meeting, and it applies to the VZ CI user too.

I also created an IAM user for the Moped test deployment and updated it on the dockerized agent. I ran a deployment of a branch, and it worked with the new credentials. I'm waiting for the Access Advisor to collect some info about what it was used for before de-scoping that one.

**Comment by mddilley** [comment]
I applied the atd-zappa policy to the test deployment IAM user and also gave permissions to only the services that the deployment uses. A test deployment succeeded with this update. Much like the VZ CI IAM user, we can reduce these further to only the actions needed by the deployment. I created https://github.com/cityofaustin/atd-data-tech/issues/11966 for this and will bring it to the next security meeting.
